tests/files/Spec.asn

```diff
@@ -0,0 +1,41 @@
+Module-name-1 DEFINITIONS AUTOMATIC TAGS ::= 
+BEGIN
+
+Identifier ::= INTEGER (0..5)
+
+Num ::= INTEGER (0..15)
+Priority ::= ENUMERATED {high, low}
+
+ASequence ::=
+ SEQUENCE {
+  priority Priority,
+  scr Identifier,
+  dst Identifier,
+  num Num,
+  length INTEGER (0..1024)
+}
+
+AChoice ::=
+ CHOICE {
+  identity Identifier,
+  num Num
+}
+
+END
+
+
+Module-name-2 DEFINITIONS AUTOMATIC TAGS ::= 
+BEGIN
+
+IMPORTS Identifier FROM Module-name-1;
+
+Id ::= Identifier
+
+SeqMod2 ::=
+ SEQUENCE {
+ isvalide BOOLEAN,
+ name BIT STRING,
```

Comment:
Should this be of type `OCTET STRING` or `IA5String` instead of `BIT STRING`? Looks like `name` is used as a string in the test case.

Reply:
yes, my bad. Feel free to change it